src/main/kotlin/no/nav/soknad/arkivering/soknadsarkiverer/config/ApplicationConfig.kt

```diff
@@ -95,7 +95,7 @@ data class AppConfiguration(val kafkaConfig: KafkaConfig = KafkaConfig(), val co
 
 	data class State(
 		var started: Boolean = false,
-		var up: Boolean = true,
+		var alive: Boolean = false,
```

Comment:
Egentlig ikke relatert til endringen, men en tekstlig forbedring

Reply:
Riktig. Det blev visst en PR på q1 istedenfor branchen `self-destruct`. Sorry for ekstra stöy